src/components/messages/PeerAndMessage/PeerAndMessage.scss

```diff
@@ -16,6 +16,12 @@
     max-width: 15em;
   }
 
+  &__text {
+    @media screen and (max-width: 700px) {
+      max-width: none;
+    }
+  }
+
```

Comment:
The preference is mobile first, can you please make this so that the default is max width is none and that the media selector is for min width instead?

Reply:
Sure thing, it keeps slipping my mind, ill go ahead and do that!